tests/component/test_channel_properties.py

```diff
@@ -0,0 +1,136 @@
+"""Contains a collection of pytest tests that validates channel properties."""
+
+from nidaqmx._task_modules.channels.ai_channel import AIChannel
+from nidaqmx._task_modules.channels.ci_channel import CIChannel
+from nidaqmx.constants import PowerIdleOutputBehavior
+
+
+def test__channel__get_boolean_property__returns_value(ai_voltage_chan_with_excit: AIChannel):
+    """Test to validate getter for channel property of boolean type."""
+    assert not ai_voltage_chan_with_excit.ai_excit_use_for_scaling
+
+
+def test__channel__set_boolean_property__returns_assigned_value(
+    ai_voltage_chan_with_excit: AIChannel,
+):
+    """Test to validate setter for channel property of boolean type."""
+    ai_voltage_chan_with_excit.ai_excit_use_for_scaling = True
+
+    assert ai_voltage_chan_with_excit.ai_excit_use_for_scaling
+
+
+def test__channel__reset_boolean_property__returns_initial_value(
```

Comment:
Change all references to "initial_value" to "default_value".

Reason: "default value" is the standard DAQmx terminology, and "initial value" could refer to a value assigned by the Create Channel function.